charts/openyurt/templates/yurt-controller-manager.yaml

```diff
@@ -127,6 +127,12 @@ rules:
       - get
       - list
       - watch
+  - apiGroups:
+      - ""
+    resources:
+      - nodes/status
+    verbs:
+      - update
```

Comment:
please also add these roles for yurt-manager component.

Reply:
I think i can close this comment. because these roles will be added for yurt-manager when @gnunu migrates this controller into yurt-manager.